data/CONTRIBUTORS.json

```
@@ -0,0 +1,14 @@
+[
+    {
+        "github_username": "zaida04",
+        "github_icon": "https://avatars.githubusercontent.com/u/19440022?v=4"
+    },
+    {
+        "github_username": "eilla1",
+        "github_icon": "https://avatars.githubusercontent.com/u/72365100?v=4"
+    },
+    {
+        "github_username": "sarthaktexas",
+        "github_icon": "https://avatars.githubusercontent.com/u/28282096?v=4"
```

you can get these by doing this:
```suggestion
        "github_icon": "https://avatars.githubusercontent.com/sarthaktexas"
```

---

by the way this means you don't have to keep this key-pair at all

---

In that case, I can just remove that key and just template it.